src/DbSchemaManager.php

```diff
@@ -40,9 +45,9 @@ final class DbSchemaManager
      */
     public function __construct(
         private ConnectionInterface $database,
-        ?string $itemsTable = null,
-        ?string $itemsChildrenTable = null,
-        ?string $assignmentsTable = null,
+        ?string $itemsTable = self::ITEMS_TABLE,
+        ?string $itemsChildrenTable = self::ITEMS_CHILDREN_TABLE,
+        ?string $assignmentsTable = self::ASSIGNMENTS_TABLE,
```

Comment:
```suggestion
        string $itemsTable = self::ITEMS_TABLE,
        string $itemsChildrenTable = self::ITEMS_CHILDREN_TABLE,
        string $assignmentsTable = self::ASSIGNMENTS_TABLE,
```

Not need nullable if use default string values.

Reply:
`null` is needed to disable creation of specific tables. What do you suggest instead?